docs/asset-managers/nfs-shares.md

```diff
@@ -1,10 +1,23 @@
 # Asset manager: `nfs-shares`
 
-The asset manager `nfs-shares` provides an asset type of the same name for resizing NFS shares
+The asset manager `nfs-shares` provides asset types for resizing NFS shares
 managed by [OpenStack Manila](https://wiki.openstack.org/wiki/Manila).
 
+* The asset type `nfs-shares` matches all Manila shares in the respective project.
+* Asset types of the form `nfs-shares-group:$NAME`, where `$NAME =~ /[A-Za-z0-9-]+/`,
+  match only those Manila shares that have the given name value in the metadata
+  key `autoscaling_group`.
+
 ## User considerations
 
+### Inter-resource constraints
+
+In each project, there can only be **either** an `nfs-shares` resource **or**
```

Comment:
```suggestion
In each project, there can only be **either** a `nfs-shares` resource **or**
```

Reply:
I double-checked to be sure, but it's correct with "an":

> The deciding factor for which of these words should be used is the sound that begins the word which follows these indefinite articles, rather than the letter which does. [...] [Words] that begin with consonants, but which are voiced as though they begin with a vowel [...] are preceded by "an", even though they begin with a consonant.

Source: https://www.merriam-webster.com/words-at-play/is-it-a-or-an